.eslintrc.json

```diff
@@ -18,7 +18,9 @@
         "react"
     ],
     "rules": {
+        "linebreak-style":0,
         "react/jsx-filename-extension": [0],
         "import/extensions": "off"
```

Comment:
We have some little changes in `.eslintrc.json` check out the [microverse-linter-config](https://github.com/microverseinc/linters-config/tree/master/react-redux), according to the project requirement we need to set the linter's correctly with the standard Microverse config file without changing it.